simulation-runner/facts/IrvingFactsChange_custom_facts.csv

```diff
@@ -0,0 +1,4801 @@
+Fact
+%22customfact1%22%3A%22vNSUclWAYnEJVu18%22%2C%22customfact2%22%3A%22jN5JseeQUdNtsILhepVR63mj%22%2C%22customfact3%22%3A%22iFfvllM0i37wjY1vBNg99FxgOpCNAbNh%22%2C
```

Comment:
Why is it necessary to encode this in the file?  Seems like it would be more readable and easier to update/maintain if we stored it not encoded and executed a method to encode it inline? Although I suppose that could also take up CPU cycles... Not sure which trade-off would be most beneficial here.

Reply:
Eventually, we will want to determine how to express "complexity" for these facts inside the json configuration file (summer intern project?) but in any case, the data should be left in the "native" format as modifying/encoding data on the fly really doesn't really buy us anything (IMO) at the risk of affecting the performance of the driver. End result is that users won't have to directly monkey with the encoded JSON, which I think everyone agrees is an eyesore and prone to mistakes.

Reply:
👍